stable/prometheus-adapter/templates/custom-metrics-apiserver-service.yaml

```diff
@@ -3,7 +3,7 @@ kind: Service
 metadata:
   annotations:
 {{ toYaml .Values.service.annotations | indent 4 }}
-  label:
+  labels:
```

Comment:
Please fix https://github.com/helm/charts/blob/98b48cb722047701477029658a4a5b78962c4d7f/stable/prometheus-adapter/templates/secret.yaml#L5 too - as was mentioned in the comments

Reply:
@Matei207 bump